app/src/main/java/com/sopt/gongbaek/di/NetworkModule.kt

```diff
@@ -0,0 +1,101 @@
+package com.sopt.gongbaek.di
+
+import android.content.Context
+import android.net.ConnectivityManager
+import com.jakewharton.retrofit2.converter.kotlinx.serialization.asConverterFactory
+import com.sopt.gongbaek.BuildConfig
+import com.sopt.gongbaek.BuildConfig.DEBUG
+import com.sopt.gongbaek.data.remote.util.NetworkUtil
+import com.sopt.gongbaek.di.qualifier.Auth
+import com.sopt.gongbaek.di.qualifier.GongBaek
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import kotlinx.serialization.json.Json
+import okhttp3.Interceptor
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import java.util.concurrent.TimeUnit
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object NetworkModule {
+    @Provides
+    @Singleton
+    fun providesOkHttpClient(
+        loggingInterceptor: HttpLoggingInterceptor,
+        @Auth authInterceptor: Interceptor
+    ): OkHttpClient =
+        OkHttpClient.Builder().apply {
+            connectTimeout(10, TimeUnit.SECONDS)
+            writeTimeout(10, TimeUnit.SECONDS)
+            readTimeout(10, TimeUnit.SECONDS)
+            addInterceptor(authInterceptor)
+            if (DEBUG) addInterceptor(loggingInterceptor)
+        }.build()
+
+    @Provides
+    @Singleton
+    fun providesJson(): Json =
+        Json {
+            isLenient = true
+            prettyPrint = true
+            explicitNulls = false
+            ignoreUnknownKeys = true
+        }
+
+    @Provides
+    @Singleton
+    fun provideLoggingInterceptor(): HttpLoggingInterceptor = HttpLoggingInterceptor().apply {
+        level = HttpLoggingInterceptor.Level.BODY
+    }
+
+    @Provides
+    @Singleton
+    @Auth
+    fun provideAuthInterceptor(): Interceptor = Interceptor { chain ->
+        val request = chain.request().newBuilder()
+            .addHeader("Authorization", "Bearer <Your_Token_Here>")
```

Comment:
[3]: 요 부분도 개인취향이긴 한데, 이런식으로 작성하면 좀 더 명시적으로 확인가능할거 같습니다!🙇🏻‍♂️
```suggestion
            .addHeader(
                name = "Authorization",
                value = "Bearer <Your_Token_Here>"
            )
```

Reply:
앱잼 내에서 로그인 구현 계획이 없어 불필요한 기능같아 삭제했습니다. 추후 참고하겠습니다!